SparkPost/2web2ui
# SE-117: UI to show a table of files uploaded in the last 10 days

### What Changed
 - Converted list results to list of list-jobs

### How To Test
 - Enable account ui option `recipientValidationV2`
 - Check that the list populates at the bottom

### To Do
